coraza.conf-recommended

```diff
@@ -138,8 +142,7 @@ SecDataDir /tmp/
 # 3:   Info
 # 4-8: Debug
 # 9:   Trace (most verbose)
-# Most logging has not been implemented because it will be replaced with
-# advanced rule profiling options
```

Comment:
It is a pretty old comment. We should maybe revisit all the logs printed in the various levels, but we have some logs for each of them, and I don't think there is some traction for `advanced rule profiling options`. I would remove these lines and not confuse the user reading it